tests/Microsoft.DotNet.Framework.Docker.Tests/projects/dotnetapp-3.5/Dockerfile

```diff
@@ -3,6 +3,9 @@ ARG BASE_RUNTIME_IMAGE
 
 
 FROM $BASE_BUILD_IMAGE as builder
```

Comment:
I realize this issue is SDK specific, but I think verifying the default SHELL more holistically would be beneficial.  We've had issues in this space in the past.  I think you could easily verify this with a shared test that executes something similar to `docker inspect mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-1909 -f "{{ .Config.Shell }}"`  I expect all images except the runtime should return the same `powershell -Command $ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';` value.

Reply:
I like that approach.